boot/bootutil/src/swap_move.c

```diff
@@ -104,8 +104,9 @@ boot_read_image_header(struct boot_loader_state *state, int slot,
 
     /* We only know where the headers are located when bs is valid */
     if (bs != NULL && out_hdr->ih_magic != IMAGE_MAGIC) {
-        rc = -1;
-        goto done;
+        /*  image is not valid, unset all flags to avoid futher
+        *  unexpected behaviour */
```

Comment:
```suggestion
         *  unexpected behaviour */
```
nit: fixing code alignment